src/main/webapp/datepicker.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+$('input[name="daterange"]').daterangepicker({
+  ranges: {
+    'Tomorrow': [moment().add(1, 'days'), moment().add(1, 'days')],
+    'Next week': [moment().startOf('week').add(1, 'weeks').add(1, 'days'), moment().startOf('week').add(2, 'weeks')],
+    'Weekend': [moment().startOf('week').add(6, 'days'), moment().startOf('week').add(7, 'days')],
```

Comment:
Note that a weekend is not defined the same way everywhere around the world. Moment.js also doesn't support querying weekends or working days since it's not really a date/time locale related information. I would probably remove this option to avoid ambiguity.

Reply:
Fair point, but I would argue that for users of countries using Monday-Friday workweek, it is a very useful feature.
According to this chart: https://en.wikipedia.org/wiki/Workweek_and_weekend#/media/File:Workweeks-map.svg, this is a large majority. Maybe I would rather just rename it to Saturday-Sunday to be precise.